query-engine/query-engine-napi/src/engine.rs

```diff
@@ -115,7 +117,10 @@ impl QueryEngine {
             .map_err(|errors| ApiError::conversion(errors, &datamodel))?
             .subject;
 
-        let flags: Vec<_> = config.subject.preview_features().map(|s| s.to_string()).collect();
+        let flags: Vec<_> = match feature_flags_overrides {
+            Some(overrides) => overrides,
+            None => config.subject.preview_features().map(|s| s.to_string()).collect(),
+        };
```

Comment:
super nitpicky — no need to change — but that can be written more compactly as `feature flags_overrides.or_else(|| config.subject.preview_features().map(|s| s.to_string()).collect()`